host/xtest/regression_1000.c

```diff
@@ -1744,3 +1746,43 @@ static void xtest_tee_test_1022(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 1022, xtest_tee_test_1022,
 		"Test dlopen()/dlsym()/dlclose() API");
+
+static void xtest_tee_test_1023(ADBG_Case_t *c)
+{
+	TEEC_Session session = {};
+	uint32_t ret_orig = 0;
+
+	Do_ADBG_BeginSubCase(c, "TPM test create session");
+	{
+		if (!ADBG_EXPECT_TEEC_SUCCESS(c,
+		    xtest_teec_open_session(&session, &tpm_log_test_ta_uuid,
+					    NULL, &ret_orig)))
+			return;
+	}
+	Do_ADBG_EndSubCase(c, "TPM test create session");
+
+	Do_ADBG_BeginSubCase(c, "TPM test service invocation");
+	{
+		TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+
+		ADBG_EXPECT_TEEC_SUCCESS(c, TEEC_InvokeCommand(&session,
+					  TA_TPM_TEST_GET_LOG,
+					  &op, &ret_orig));
+	}
+	Do_ADBG_EndSubCase(c, "TPM test service invocation");
+
+	Do_ADBG_BeginSubCase(c, "TPM test passing short buffer");
+	{
+		TEEC_Operation op = TEEC_OPERATION_INITIALIZER;
+
+		ADBG_EXPECT_TEEC_SUCCESS(c, TEEC_InvokeCommand(&session,
+					  TA_TPM_TEST_SHORT_BUF,
+					  &op, &ret_orig));
+	}
+	Do_ADBG_EndSubCase(c, "TPM test passing short buffer");
+
+	TEEC_CloseSession(&session);
+}
+
+ADBG_CASE_DEFINE(regression, 1023, xtest_tee_test_1023,
```

Comment:
Adding a new test case is fine. But, I have already used 1023 in https://github.com/OP-TEE/optee_test/pull/394.
We'll see what gets merged first.

Reply:
NP, I will leave it untouched and change it later if there were any conflict.

Reply:
Fine.